app/views/steps/shared/_relationship_step.html.erb

```diff
@@ -1,17 +1,15 @@
 <% heading = t('.heading', person_name: form_object.record.person.full_name, child_name:  form_object.record.minor.full_name) %>
 
-<h1 class="heading-xlarge gv-u-heading-xxlarge">
-  <%=  heading %>
-</h1>
+<h1 class="govuk-heading-xl"><%=  heading %></h1>
 
-<%= step_form form_object do |f| %>
-  <%=
-    f.radio_button_fieldset :relation, { legend_options: { class: 'visually-hidden', text: heading } } do |fieldset|
-      fieldset.radio_input Relation::MOTHER
-      fieldset.radio_input Relation::FATHER
-      fieldset.radio_input(Relation::OTHER) { f.text_field :relation_other_value }
-    end
-  %>
+<%= step_form form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+  <%= f.govuk_radio_buttons_fieldset :relation do %>
+    <%= f.govuk_radio_button :relation, Relation::MOTHER, link_errors: true %>
```

Comment:
Only the first radio needs the `link_errors` flag, the others can be removed as don't have any effect.

Reply:
I was under the impression from looking at the validation on staging (for comparison) that it needs to validate the input of the text field when selecting `OTHER` since it throws a validation error without nothing on the text field?

Reply:
Text fields are always linked automatically as they have an unique ID so don't need that.

Radio fieldsets or Checkbox fieldsets produce an error on the whole fieldset, so in order to auto-focus and link properly to the radio or checkbox we want, we need to mark it with `link_errors: true`